printer.cfg

```diff
@@ -11,13 +11,13 @@
 [include config/hardware/*.cfg]
 
 ###########################
-### SOFTWARE COMPONENTS ###
+### FUNCTIONS COMPONENTS ###
 ###########################
-[include config/software/bed_mesh.cfg]
-[include config/software/firmware_rectraction.cfg]
-[include config/software/input_shaper.cfg]
-[include config/software/qgl.cfg]
-[include config/software/z_calibration.cfg]
+[include config/functions/bed_mesh.cfg]
+[include config/functions/firmware_rectraction.cfg]
+[include config/functions/input_shaper.cfg]
+[include config/functions/qgl.cfg]
+[include config/functions/z_calibration.cfg]
```

Comment:
As said in the general review comment. Please keep `config/software` for the moment.